plugin.xml

```diff
@@ -80,7 +80,8 @@
     <!-- android -->
     <platform name="android">
 
-        <framework src="com.android.support:support-v4:+" value="gradle" />
+        <!--<framework src="com.android.support:support-v4:+" value="gradle" /> -->
+        <framework src="src/android/local-notifications-build.gradle" custom="true" type="gradleReference" />
```

Comment:
Wrong place. We have been using the path libs/android for gradle extras. Check [here](https://github.com/OutSystems/cordova-outsystems-logger/tree/master/libs/android) for an example. The file name I leave to your discretion. Also remove the commented line above?